diff --git a/panels/meson.build b/panels/meson.build
index 2f4fdc5e3..2fc05a29f 100644
--- a/panels/meson.build
+++ b/panels/meson.build
@@ -16,7 +16,6 @@ panels = [
   'microphone',
   'mouse',
   'notifications',
-  'online-accounts',
   'power',
   'printers',
   'region',
diff --git a/shell/cc-panel-loader.c b/shell/cc-panel-loader.c
index f20384394..771df34e2 100644
--- a/shell/cc-panel-loader.c
+++ b/shell/cc-panel-loader.c
@@ -48,7 +48,7 @@ extern GType cc_network_panel_get_type (void);
 extern GType cc_wifi_panel_get_type (void);
 #endif /* BUILD_NETWORK */
 extern GType cc_notifications_panel_get_type (void);
-extern GType cc_goa_panel_get_type (void);
+/* extern GType cc_goa_panel_get_type (void); */
 extern GType cc_power_panel_get_type (void);
 extern GType cc_printers_panel_get_type (void);
 extern GType cc_region_panel_get_type (void);
@@ -112,7 +112,7 @@ static CcPanelLoaderVtable default_panels[] =
   PANEL_TYPE("wifi",             cc_wifi_panel_get_type,                 cc_wifi_panel_static_init_func),
 #endif
   PANEL_TYPE("notifications",    cc_notifications_panel_get_type,        NULL),
-  PANEL_TYPE("online-accounts",  cc_goa_panel_get_type,                  NULL),
+  /* PANEL_TYPE("online-accounts",  cc_goa_panel_get_type,                  NULL), */
   PANEL_TYPE("power",            cc_power_panel_get_type,                NULL),
   PANEL_TYPE("printers",         cc_printers_panel_get_type,             NULL),
   PANEL_TYPE("region",           cc_region_panel_get_type,               NULL),
