diff --git a/test/Makefile b/test/Makefile
index 2751f67..6f7d9cb 100644
--- a/test/Makefile
+++ b/test/Makefile
@@ -41,7 +41,6 @@ test_targets += \
 	d4ae271dfaae-test \
 	d77a67ed5f27-test \
 	defer \
-	double-poll-crash \
 	eeed8b54e0df-test \
 	eventfd \
 	eventfd-disable \
@@ -99,7 +98,6 @@ test_targets += \
 	sq-poll-dup \
 	sq-poll-kthread \
 	sq-poll-share \
-	sqpoll-disable-exit \
 	sqpoll-exit-hang \
 	sqpoll-sleep \
 	sq-space_left \
@@ -175,7 +173,6 @@ test_srcs := \
 	d4ae271dfaae-test.c \
 	d77a67ed5f27-test.c \
 	defer.c \
-	double-poll-crash.c \
 	eeed8b54e0df-test.c \
 	eventfd-disable.c \
 	eventfd-ring.c \
@@ -233,7 +230,6 @@ test_srcs := \
 	sq-poll-dup.c \
 	sq-poll-kthread.c \
 	sq-poll-share.c \
-	sqpoll-disable-exit.c \
 	sqpoll-exit-hang.c \
 	sqpoll-sleep.c \
 	sq-space_left.c \
